Dockerfile.train.tmpl

```diff
@@ -1,6 +1,7 @@
 # Please refer to the TRAINING documentation, "Basic Dockerfile for training"
 
 FROM tensorflow/tensorflow:1.15.2-gpu-py3
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
I thought we couldn't have ARG due to some old version of Docker? This is breaking CI: https://community-tc.services.mozilla.com/tasks/fG_UDYcmQJCgx_J54PNs5w/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FfG_UDYcmQJCgx_J54PNs5w%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

Reply:
I'm changing it to ENV in #3085